tests/test_apps/config_source_test/structured.py

```diff
@@ -12,6 +12,11 @@ class ConfigWithoutGroup:
     group: bool = False
 
 
+@dataclass
+class ConfigWithUnicode:
+    数据库: str = "mysql"
```

Comment:
eeek.
can you switch to unicode in the value instead of the key?
I know it's not exactly the bug report but it should be fine.